app/concepts/api/v1/users/registrations/operation/create.rb

```diff
@@ -31,7 +31,11 @@ def create_email_token(ctx)
 
     def send_email(ctx)
       user = ctx[:user]
-      RegistrationMailer.with(user: user, email_token: ctx[:email_token]).confirmation_email.deliver_later
+      RegistrationMailer.confirmation_email(
```

Comment:
```ruby
Success(RegistrationMailer.confirmation_email...
```

Reply:
But controller expects from operation to return User entity, doesn't it?